pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -44,6 +44,16 @@ If you have an HDMI screen attached, you should see balena logo on the screen wh
 
 If you still can't get your device online, come on over and talk to us on our [support channel][usingSupport].
 
+### This is the wrong balena device.
+
+If you see this error, there are several potential causes:
+- The config.json file is missing
+- The UUID in the config.json file does not match the device's UUID
+  - This could be caused by config.json corruption or storage corruption
+- `jq` failed
```

Comment:
It's a command but I don't know enough about this bit to elaborate 😓 cc @lmbarros can you please help out with this one since you raised this potential cause?

Reply:
I know about the command but why folks be using it, and what context?

Reply:
idk, I think it's an internal usage. Need to defer to Leandro tbh

Reply:
Yeah, internal usage, `jq` is used in the command that can trigger this error message to appear. I agree it's a bad idea to mention it explicitly in the docs, as it won't make sense to the reader.

I was trying to come up better wording for that, but everything I can think of is so generic that it doesn't help at all. Like "a command failed"... this doesn't really inform or helps the user to solve the issue. Maybe just leave this out?